app/controllers/concerns/inactive_notifier.rb

```diff
@@ -0,0 +1,21 @@
+module InactiveNotifier
```

Comment:
Should we stick with the Rails naming convention and call it `InactiveNotifiable`? I guess would also need to update the other concerns `TrialNotfiable`. Up to you.

Reply:
Yes good point, I'll rename it